.github/workflows/format.yml

```
@@ -0,0 +1,26 @@
+name: clang-format
+
+on:
+  push:
+
+jobs:
+  format:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+      with:
+        token: ${{ secrets.GITHUB_TOKEN }}
```

```suggestion
    - uses: actions/checkout@v3
      with:
        token: ${{ secrets.GITHUB_TOKEN }}
        ref: ${{ github.ref }}
```

こんなんで fatal: You are not currently on a branch. 解消されたりしないですかね

---

あれ、clang-format ってどこで回っているんだ

---

https://github.com/luzhiled1333/comp-library/actions

ここに出てこなくない？これ

---

あ、わかった　ここか　https://github.com/ei1333/comp-library/actions